2d/platformer/src/titlescreen/TitleScreen.gd

```diff
@@ -0,0 +1,20 @@
+extends Node
+
+
+
+func _ready():
+	$MarginContainer/VBoxContainer/VBoxContainer/TextureButton.grab_focus()
+
+func _physics_process(delta):
+	if $MarginContainer/VBoxContainer/VBoxContainer/TextureButton.is_hovered() == true:
+		$MarginContainer/VBoxContainer/VBoxContainer/TextureButton.grab_focus()
+	if $MarginContainer/VBoxContainer/VBoxContainer/TextureButton2.is_hovered() == true:
```

Comment:
small nit. You don't need the == true

Reply:
I would recommend using ( ) around both "if" statements. Other than that it looks good